src/components/Tinder/Tinder.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component} from 'react';
+import getStore from './../../utils/honestyStore';
+import {ControllerDataset} from '../Admin/Trainer/ControllerDataset';
+
+class Tinder extends Component {
+  state = {
+    loading: true,
+    image: null
+  };
+
+  nextImage = async () => {
+    return this.controllerDataset
+      .getUntrustedImage()
+      .then(image => this.setState({image}));
+  };
+
+  trustImage = () => {
+    this.controllerDataset
+      .trustImage({id: this.state.image.id, item: this.state.image.label})
+      .then(this.nextImage);
+  };
+
+  setAsUnknown = () => {
+    this.controllerDataset
+      .setLabel(this.state.image.id, 'unknown')
+      .then(this.nextImage);
+  };
+
+  deleteImage = () => {
+    this.controllerDataset
+      .deleteImage({id: this.state.image.id, item: this.state.image.label})
+      .then(this.nextImage);
+  };
+
+  changeCategory = event => {
+    this.controllerDataset
+      .setLabel(
+        this.state.image.id,
+        Object.values(this.storeList).find(
```

Comment:
Is the store list mapped `{ [id]: item }` where `item` also has `item.id`?

If so could we search the keys (ids) to find the one we want?

e.g.

```
Object.keys(this.storeList).find(id => this.storeList[id].name === event.target.value)
```